### DISCUSSION TITLE
Version 5.31.3 not completely published?

### DISCUSSION THREAD

**Discussion by simschla** [dcm4che/dcm4che#1390] [Q&A] [answered]
Hi

I noticed there is a Version `5.31.3` for some of the jars, e.g. org.dcm4che:dcm4che-core [here](https://dcm4che.org/maven2/org/dcm4che/dcm4che-core/5.31.3/) but **not** for other modules, e.g. org.dcm4che:dcm4che-dcmr [here](https://dcm4che.org/maven2/org/dcm4che/dcm4che-dcmr/).
Also, the Releases-Page on github shows **no** `5.31.3` Version...

Is there a problem with 5.31.3 or a reason why it is not "fully" published?

As it is right now, it confuses our dependency-bot (renovate) as it cannot determine a correct version to upgrade to.

**Comment by gunterze** [comment, marked as answer]
Last release is 5.31.2. 5.31.3 versions for all modules will be uploaded on release of 5.31.3. Already uploaded jars for 5.31.3 contains  current snapshot version, needed to build docker image [dcm4che/keycloak:23.0.4](https://hub.docker.com/repository/docker/dcm4che/keycloak/general).

**Reply by simschla** [reply]
So those should have been `-SNAPSHOT` releases for correct maven version handling? Thanks for clarification.
When is releasing the next "full" version planned, so our renovate run is "deconfused"?

**Comment by simschla** [comment]
@gunterze Same thing again with 5.32.0?

**Reply by simschla** [reply]
And as far as I can see, the version 5.31.3 is still not fully released, correct @gunterze ? 

May I suggest you use either snapshot-releases or a local maven repo for the Snapshot-Releases required for the docker-image? The way it is handled now is pretty confusing to people outside the project 😞  And in addition our dependency automation tools (dependabot/renovate) fail to handle the dcm4che library with the current versioning approach 😞
